framework/areg/base/ThreadAddress.hpp

```diff
@@ -260,10 +260,16 @@ inline String ThreadAddress::convToString(void) const
 //////////////////////////////////////////////////////////////////////////
 inline const IEInStream & operator >> (const IEInStream & stream, ThreadAddress & input)
 {
-    return ( stream >> input.mThreadName );
+    //FIXME
```

Comment:
This still needs some improvements, I think

Reply:
Hi @biwiki 
Can you please instead of `//FIXME` write
```cpp
//FIXME
// return ( stream >> input.mThreadName );
```
That I know what exactly it was and what exactly should be fixed.

Reply:
It is fixed now, thanks @aregtech